### ISSUE TITLE
command ~$ vue add unit-mocha shows 404 error for an existing vue cli 3 project

### ISSUE THREAD

**Issue by wasifsn** [vuejs/vue-cli#4793]
### Version
3.10.0

### Environment info
```
Environment Info:

  System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
  Binaries:
    Node: 10.16.0 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.12.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.387.0
  npmPackages:
    @saeris/vue-spinners: ^1.0.8 => 1.0.8
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0
    @vue/babel-plugin-transform-vue-jsx:  1.0.0
    @vue/babel-preset-app:  3.10.0
    @vue/babel-preset-jsx:  1.1.0
    @vue/babel-sugar-functional-vue:  1.0.0
    @vue/babel-sugar-inject-h:  1.0.0
    @vue/babel-sugar-v-model:  1.0.0
    @vue/babel-sugar-v-on:  1.1.0
    @vue/cli-overlay:  3.10.0
    @vue/cli-plugin-babel: ^3.10.0 => 3.10.0
    @vue/cli-plugin-eslint: ^3.10.0 => 3.10.0
    @vue/cli-plugin-unit-mocha: ^4.0.5 => 4.0.5
    @vue/cli-service: ^3.10.0 => 3.10.0
    @vue/cli-shared-utils:  3.10.0 (4.0.5)
    @vue/component-compiler-utils:  2.6.0 (3.0.1)
    @vue/preload-webpack-plugin:  1.1.1
    @vue/test-utils: ^1.0.0-beta.29 => 1.0.0-beta.29
    @vue/web-component-wrapper:  1.2.0
    babel-helper-vue-jsx-merge-props: ^2.0.3 => 2.0.3
    eslint-plugin-vue: ^5.0.0 => 5.2.3 (4.7.1)
    vue: ^2.6.10 => 2.6.10
    vue-eslint-parser:  2.0.3 (5.0.0)
    vue-hot-reload-api:  2.3.4
    vue-jest: ^3.0.5 => 3.0.5
    vue-loader: ^15.7.2 => 15.7.2
    vue-loading-overlay: ^3.2.0 => 3.2.0
    vue-loading-spinner: ^1.0.11 => 1.0.11
    vue-multiselect: ^2.1.6 => 2.1.6
    vue-nav-tabs: ^0.5.7 => 0.5.7
    vue-router: ^3.1.3 => 3.1.3
    vue-select: ^3.1.0 => 3.1.0
    vue-style-loader:  4.1.2
    vue-template-compiler: ^2.6.10 => 2.6.10
    vue-template-es2015-compiler:  1.9.1
    vuejs-dialog: ^1.4.1 => 1.4.1
    vuelidate: ^0.7.4 => 0.7.4
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
run command  ~$ vue add unit-mocha  in an existing project

### What is expected?
the plugin should be installed 

### What is actually happening?
$ vue add unit-mocha

�📦  Installing vue-cli-plugin-unit-mocha...

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/vue-cli-plugin-unit-mocha - Not found
npm ERR! 404
npm ERR! 404  'vue-cli-plugin-unit-mocha@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Piktorlabs\AppData\Roaming\npm-cache\_logs\2019-11-04T11_12_02_352Z-debug.log
 ERROR  command failed: npm install --loglevel error -D vue-cli-plugin-unit-mocha

---
please fix the server if thats the case

**Comment by sodatea** [comment]
Please update your global `@vue/cli` version. It's already fixed since v3.11.0

**Comment by wasifsn** [comment]
Thanks a lot sodatea !!
cheers!!
